libmozevent/utils.py

```diff
@@ -146,6 +146,15 @@ class AsyncRedis(object):
     async def connect():
         if AsyncRedis.redis.get(None) is None:
             AsyncRedis.redis.set(
-                await aioredis.from_url(os.environ["REDIS_URL"], decode_responses=False)
+                await aioredis.from_url(
+                    os.getenv("REDIS_TLS_URL", None),
+                    decode_responses=False,
+                    ssl_cert_reqs=None,
+                )
+                if os.getenv("REDIS_TLS_URL", None) is not None
+                else await aioredis.from_url(
+                    os.getenv("REDIS_URL", None),
```

Comment:
Nit: if we're here REDIS_URL must be there, so you can just do `os.environ["REDIS_URL"]`